docs/components/modeler/desktop-modeler/plugins/plugins.md

```diff
@@ -76,28 +93,30 @@ Plug them into the modeler like this:
 
 ```javascript
 module.exports = {
-  menu: './menu-entries'
+  menu: './menu-entries'
 };
 ```
 
-You can use a Node.js module here since the modeler is built with [Electron](https://electron.atom.io/) which uses [Node.js](https://nodejs.org/en/).
+> The code within the menu entries is special because it executes on [the main process](https://www.electronjs.org/docs/latest/tutorial/process-model) of Electron. This comes with the advantage of allowing you to use [Node.js](https://nodejs.org/en/) modules, and the disadvantage of not being able to debug your code in Chromium. For more information regarding main process debugging, please refer to the [official Electron documentation](https://www.electronjs.org/docs/latest/tutorial/debugging-main-process).
```

Comment:
@MaxTru Added this entry, thoughts?

Reply:
I like. 

I would propose a slight rephrasing (not a disadvantage, but a challenge to consider ;-) ). Something like:

```suggestion
> The code within the menu entries executes on [the main process](https://www.electronjs.org/docs/latest/tutorial/process-model) of Electron. This comes with the advantage of allowing you to use [Node.js](https://nodejs.org/en/) modules, but you need to consider, that you can not debug the respective code in Chromium. For more information regarding main process debugging, please refer to the [official Electron documentation](https://www.electronjs.org/docs/latest/tutorial/debugging-main-process).
```